### DISCUSSION TITLE
project Molecular Dynamics trajectory onto a set of phonons

### DISCUSSION THREAD

**Discussion by Satinelamp** [deepmodeling/abacus-develop#2287] [Ideas]
### Background

After running MD of a supercell or a cluster, we'd like to know the excitation of different phonons, it would be helpful if Abacus could project the Molecular Dynamics (MD) trajectory and projects it onto a set of harmonic phonon modes obtained by [Phonopy](http://phonopy.sourceforge.net/).

### Describe the solution you'd like

I've seen some papers use the method called normal mode decomposition.

https://www.sciencedirect.com/science/article/abs/pii/S0008622315005230
https://wulixb.iphy.ac.cn/pdf-content/10.7498/aps.63.154704.pdf

**Comment by dyzheng** [comment]
Maybe this tutorial documentation can help you: 
[ABACUS+Phonopy计算声子谱.pdf](https://github.com/deepmodeling/abacus-develop/files/11270266/ABACUS%2BPhonopy.pdf)
